src/payout_queue/entity.rs

```diff
@@ -1,12 +1,13 @@
+use super::config::*;
+use crate::primitives::*;
```

Comment:
top level should be reserved for external dependencies. 

then crate level

then super

something like 
```
use derive_builder;

use crate::primitives;

use super::config::*;
```